### ISSUE TITLE
Contrast of text not sufficient (WCAG 1.4.3a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1227]
1. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 

- The link texts on pages 9 and 10 don´t meet the required contrast of 4.5: 1. It´s recommendable to use the following color for the link text: # 003366.

2. https://slidewiki.org/addDeck 

- The link text "Hilfspräsentationen" does not achieve the required contrast of 4.5: 1 to the background.
- The link text "Hilfspräsentationen" does not achieve the required contrast of 3: 1 to the surrounding text. (When underlining the text, this contrast requirement is not necessary.)

3. https://slidewiki.org/contactus 

- The text within the confirmation button (error message) does not have the required contrast of 4.5:1. (By uniformly adapting the error messages within SlideWiki, this note may be not relevant.)

![grafik](https://user-images.githubusercontent.com/48785136/73826864-cc1ded00-47fe-11ea-9489-d2ee9b2170fd.png)
